### ISSUE TITLE
Open share links in new tab / new window

### ISSUE THREAD

**Issue by schulle4u** [GiovanniSalmeri/yellow-minishare#2]
Currently all share links are opened in the same window, unless the visitor of a site decides to manually open it in a new tab or new window. If the visitor clicks a share link and want to continue reading the website after sharing, he has to go back first which is a little inconvenient. Maybe a `target=_blank` should be added to the link? 

Steffen

**Comment by GiovanniSalmeri** [comment]
Solved with <https://github.com/GiovanniSalmeri/yellow-minishare/commit/c1b5a03876e81f527aa87bf38bd2cd8e0f0e4d8e>

In general I think that the user should choose whether a link should open in a different tab/window (in HTML 4 / XHTML 1 strict, the target attribute was entirely suppressed)... but I checked several types of Social buttons and *all* open the links in a popup window. So you are evidently right! The only exception I was able to find is <http://nngroup.com> (which was my primary inspiration: the plain style of Minishare is shamelessly copied from there). So I introduced a tiny javascript file that changes the behaviour of the links. Just in case there is also a new setting `MinishareSamePage` (default 0).
